itl/plugins-contrib.d/smart-advanced.conf

```diff
@@ -0,0 +1,63 @@
+/* 
+ * Icinga2 CheckCommand definition for the SMART Advanced (check_smart.pl) Monitoring Plugin
+ */
+
+object CheckCommand "smart-advanced" {
+	command = [ PluginContribDir + "/check_smart.pl" ]
+
+	arguments = {
+		"-d" = {
+			value = "$smart_device$"
+			set_if = {{ macro("$smart_device_is_glob$") == false }}
+			description = "A physical block device to be SMART monitored, eg /dev/sda."
+		}
+		"-g" = {
+			value = "$smart_glob$"
```

Comment:
I'd also use `smart_device` here. This would have the advantage that you can't build ambiguous configuration like
```
vars.smart_device = "/dev/sda"
vars.smart_glob = "/dev/sd[b-z]"
```
but instead would either do
```
vars.smart_device = "/dev/sda"
```
or
```
vars.smart_device = "/dev/sd[b-z]"
vars.smart_device_is_glob = true
```

When doing this, also remember to update the documentation accordingly.

Reply:
I think this is a very smart suggestion, which deals perfectly with the fact, that you can either use `-d`or `-g`

Reply:
https://gist.github.com/waja/6d5e33c814a20d4834ac47adba92ee24 works very well for me

Reply:
Updated in latest commit.

Reply:
If OK, can you use "squash and merge" from your/repo side please?